addons/mrp_account/wizard/post_wip.py

```diff
@@ -0,0 +1,120 @@
+    def _compute_credit(self):
+        for record in self:
+            if record.debit != 0:
+                record.credit = 0
+
+
+class PostWip(models.TransientModel):
+    _name = 'mrp.account.post.wip'
+    _description = 'Post Manufacturing WIP'
+
+    @api.model
+    def default_get(self, fields_list):
+        res = super().default_get(fields_list)
+        if 'date' in fields_list:
+            res['date'] = fields.Datetime.from_string(self.env.context.get('date'))
+            res['reversal_date'] = res['date'] + timedelta(days=1)
+        if 'journal_id' in fields_list:
+            default = self.env['ir.property']._get_default_property('property_stock_journal', 'product.category')[1]
+            if default:
+                res['journal_id'] = default[1]
+        if 'reference' in fields_list:
+            res['reference'] = _('Manufacturing - WIP')
+        if 'line_ids' in fields_list:
+            res['line_ids'] = self._generate_lines()
+        return res
+
+    date = fields.Date("Date")
+    reversal_date = fields.Date("Reversal Date")
+    journal_id = fields.Many2one('account.journal', "Journal")
+    reference = fields.Char("Reference")
+    line_ids = fields.One2many('mrp.account.post.wip.line', 'post_wip_id', "Post WIP lines")
+
+    def _get_overhead_account(self):
+        overhead_account = self.env.company.account_production_wip_overhead_account_id
+        if overhead_account:
+            return overhead_account.id
+        cop_acc = self.env['ir.property']._get_default_property('property_stock_account_production_cost_id', 'product.category')[1]
+        if cop_acc:
+            return cop_acc[1]
+        return self.env['ir.property']._get_default_property('property_stock_account_input_categ_id', 'product.category')[1]
+
+    def _generate_lines(self):
+        productions = self.env['mrp.production'].search([('state', 'in', ('progress', 'to_close', 'confirmed'))])
+        compo_value = -sum(
+            move.product_id._prepare_out_svl_vals(move.quantity, None)['value']
+            for move in productions.move_raw_ids.filtered(lambda m: m.picked and m.quantity)
+        )
+        overhead_value = productions.workorder_ids._cal_cost()
+        sval_acc = self.env['ir.property']._get_default_property('property_stock_valuation_account_id', 'product.category')[1]
+        sval_acc = sval_acc[1] if sval_acc else False
+        return [
+            (0, 0, {
+                'label': _('WIP - Component Value'),
+                'credit': compo_value,
+                'account_id': sval_acc,
+            }),
+            (0, 0, {
+                'label': _('WIP - Overhead'),
+                'credit': overhead_value,
+                'account_id': self._get_overhead_account(),
+            }),
+            (0, 0, {
+                'label': _('Manufacturing - WIP'),
+                'debit': compo_value + overhead_value,
+                'account_id': self.env.company.account_production_wip_account_id.id,
```

Comment:
I just checked in with mgm and he says that for this one it should not have any fallbacks, i.e. when you set it to empty in the settings it can just be empty in the wizard for the user to set manually